packages/analytics/analytics-chart/sandbox/pages/TimeSeriesChartDemo.vue

```diff
@@ -357,7 +357,7 @@ const analyticsChartOptions = computed<AnalyticsChartOptions>(() => {
   return {
     type: chartType.value,
     stacked: stackToggle.value,
-    chartDatasetColors: colorPalette.value,
+    // chartDatasetColors: colorPalette.value,
```

Comment:
just make sure to remove this commented out line, and the one above. I assume the fallbacks are now working as expected.